src/Runner.Common/Util/EncodingUtil.cs

```diff
@@ -0,0 +1,50 @@
+using GitHub.Runner.Sdk;
+using GitHub.Runner.Common;
+
+namespace GitHub.Runner.Common.Util
+{
+    using System;
+
+    public static class EncodingUtil
+    {
+        public static async void SetEncoding(IHostContext hostContext, Tracing trace, System.Threading.CancellationToken cancellationToken)
```

Comment:
typically `async Task SetEncoding` technically void works but history... intention was for GUI back compat.